packages/libs/react-ui/src/components/Link/index.ts

```diff
@@ -1,17 +1,8 @@
 import type { ILinkProps } from './Link';
 import { Link as LinkRoot } from './Link';
```

Comment:
I think it's better to rename the files so there is no reference to Root anymore. Should be able to make this file much smaller as well